src/components/Button.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import Button from '@mui/material/Button';
+
+export default function CustomButton(props) {
+  return (
+    <Button variant="text">{props.title}</Button>
```

Comment:
Good way of using Material UI button, but we can add more props to make it flexible